src/ontology/cl-edit.owl

```diff
@@ -29470,6 +29471,21 @@ SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
 
+# Class: obo:CL_4023122 (oxytocin receptor sst GABAergic cortical interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) obo:IAO_0000115 obo:CL_4023122 "An interneuron located in the cerebral cortex that express the oxytocin receptor. These interneurons also express somatostatin.")
```

Comment:
@shawntanzk sorry for not seeing it sooner: An interneuron ... expresses (instead of express)